### PR TITLE
Minor refactor to merge similar methods into one:

### PR DESCRIPTION
**Pull request by nguyen-phillip**
* SNTSyncdQueue addBundleEvents: and addEvent: became addEvents:isFromBundle:
* SNTSyncdQueue backoffForBundleHash: and backoffForEvent: became backoffForPrimaryHash:
* SNTCommandSyncManager postBundleEventsToSyncServer: and postEventToSyncServer: became postEventsToSyncServer:isFromBundle:

### CODE REVIEW

File: Source/santad/SNTSyncdQueue.m [hunk 1]
```diff
@@ -38,33 +38,30 @@ - (instancetype)init {
   return self;
 }
 
-- (void)addBundleEvents:(NSArray<SNTStoredEvent *> *)events {
-  if (![self backoffForBundleHash:events.firstObject.fileBundleHash]) return;
+- (void)addEvents:(NSArray<SNTStoredEvent *> *)events isFromBundle:(BOOL)isFromBundle {
+  if (!events || events.count == 0) return;
```
Comment (tburgin) [inline Source/santad/SNTSyncdQueue.m]: nit: if (!events || !events.count) return;

Comment (nguyen-phillip) [reply Source/santad/SNTSyncdQueue.m]: fixed

Comment (russellhancox) [reply Source/santad/SNTSyncdQueue.m]: You can avoid the first condition altogether

Comment (nguyen-phillip) [reply Source/santad/SNTSyncdQueue.m]: oops, done.

**Pull request merged**